src/ol_infrastructure/substructure/keycloak/Pulumi.substructure.keycloak.QA.yaml

```diff
@@ -37,6 +37,7 @@ config:
       mitlearn: ["https://api.rc.learn.mit.edu/*", "https://rc.learn.mit.edu/*"]
       open: ["https://api.mitopen-rc.odl.mit.edu/*", "https://mitopen-rc.odl.mit.edu/*"]
       open-discussions: ["https://discussions-rc.odl.mit.edu/*"]
+    extra_default_scopes: ["ol-profile"]
```

Comment:
@shaidar I feel like this would be better done as a property under `client_info`, but that'd involve refactoring that to be an object w/ `urls` which is probably more invasive here and better of as a separate refactor.